ClosedXML/Excel/DataValidation/XLDataValidations.cs

```diff
@@ -254,7 +254,10 @@ private void ProcessRangeRemoved(IXLRange range)
             var entry = _dataValidationIndex.GetIntersectedRanges((XLRangeAddress) range.RangeAddress)
                 .SingleOrDefault(e => Equals(e.RangeAddress, range.RangeAddress));
 
-            _dataValidationIndex.Remove(entry.RangeAddress);
```

Comment:
This `entry.RangeAddress` right after `.SingleOrDefault` looks like a time bomb waiting for NRE.
 I suspected there must be some explanation and sure enough - before 5fe157f18b40a90a5524812fa1f358a2d5ca0b2c the entire `entry` was passed into `Remove` which handled null by itself. Then, during refactoring, the `.RangeAddress` was mechanically added without too much attention and 💥 .